shared/rocket/build.gradle.kts

```diff
@@ -1,19 +1,54 @@
+import org.jetbrains.kotlin.gradle.plugin.mpp.apple.XCFramework
+
 plugins {
     alias(libs.plugins.quanti.kmp.library)
     alias(libs.plugins.quanti.android.detekt)
     alias(libs.plugins.quanti.android.ktlint)
     alias(libs.plugins.kover)
+    alias(libs.plugins.kotlin.multiplatform)
```

Comment:
This is extra, already applied within convention plugin `build-logic/convention/src/main/kotlin/plugins/KmpLibConventionPlugin.kt`